lib/stdlib/src/erl_scan.erl

```diff
@@ -425,6 +425,12 @@ scan1("."=Cs, St, Line, Col, Toks) ->
     {more,{Cs,St,Col,Toks,Line,[],fun scan/6}};
 scan1([$.=C|Cs], St, Line, Col, Toks) ->
     scan_dot(Cs, St, Line, Col, Toks, [C]);
+scan1([$",$",$"|Cs], St, Line, Col, Toks) -> %" Emacs
```

Comment:
Is that documented somewhere?  In the file I find 14 with random column and one section with 9 in column 48 (0-based).  I have always interpreted them to be used after code.

Reply:
My take is also that they should be used after code, but the emacs formatting is documented as indenting single `%` to `comment-column` columns (48 by default), which this doesn't follow.

We try to our best to keep everything compiler-related emacs-formatted, so it might be better to change these to `%%` or move them to the default column for `%`.

> The main feature of Erlang mode is indentation, press TAB and the current line will be indented correctly.
>
>Comments starting with only one `%' are indented to the column stored in the variable `comment-column'.  Comments starting with two `%':s are indented with the same indentation as code. Comments starting with at least three `%':s are indented to the first column.

Reply:
I misremembered: the formatter doesn't change it, it's just the compiler style that I'm apparently far too used to. :smile:

I take it all back, feel free to keep it this way. :)

Reply:
I'll keep it just because the closest surrounding code already has the same style, but I *do* note that the function `escape_char/1` in the same file follows the "compiler style" you are used to... :-/